CHANGELOG.md

```diff
@@ -1,4 +1,12 @@
+## 1.5.1
```

Comment:
I think I like this better as a 'z' than as a 'y', but, worth raising the question.